doc/whats_new/v0.20.rst

```diff
@@ -68,6 +72,13 @@ Linear, kernelized and related models
 Bug fixes
 .........
 
+Ensemble
+
+- Fix allowing to obtain deterministic with :class:`BaseBagging` estimator,
+  when comparing results generated at fit time with the one using the object
+  attributes when ``random_state`` is set. :issue:`9524` by :user:`Guillaume
```

Comment:
please use the PR number in what's new. This helps diffing the commit history against the changelog